packages/client/modules/meeting/components/TemplateScaleValueList.tsx

```diff
@@ -76,16 +78,18 @@ const TemplateScaleValueList = (props: Props) => {
             )
           }}
         </Droppable>
-      </ScaleList>
-    </DragDropContext>
+      </DragDropContext>
+      <AddScaleValueButtonInput scale={scale} />
```

Comment:
Putting this button/input in the list means it feels like an item when it's an input